assets/src/util/assignment.js

```diff
@@ -232,7 +232,14 @@ const createUserSettings = (courseId, viewName, setting) => {
   return mutation
 }
 
+const assignmentStatus = {
+  GRADED: 'Graded',
+  SUBMITTED: 'Not Yet Graded',
+  UNSUBMITTED: 'Unsubmitted'
+}
```

Comment:
Can you apply the change to the `AssignmentPlanningV2.js` file as well? since this code can be reused their. Should be simple Change!

Reply:
Yeah, thanks I forgot that.